src/index.js

```diff
@@ -86,7 +86,11 @@ function handlebars(options) {
 
       for (var partial of scanner.partials) {
         // Register the partial dependencies as partials.
-        body += `import '${partial}${options.templateExtension}';\n`;
+        // do not import special partial @partial-block: http://handlebarsjs.com/partials.html#partial-block
+        if (!partial.startsWith('@')) {
+          // rollup-plugin-root-import require import from '/'
```

Comment:
Heh, I haven't thought of that simple solution - putting slash in template path...

I'am introducing rollup to existing project and we have hundreds of templates in example form: `{{#> dropdown ...}}` so I've totally rejected the idea of changing every template..

Now I've understand your concept. I thing you should consider updating docs by putting simple example  of using '/' in partial inject ('{{> '/views/_loading'}}') - it could help others not to stuck on partials.

Thanks for your response, I will think about best solution for my project (changing all templates or something ?). First simple idea is to add a configuration parameter for steering '/' inclusion, I could add another PR for that.